pkg/engine/const.go

```diff
@@ -113,7 +113,6 @@ const (
 	kubernetesMasterGenerateProxyCertsScript = "k8s/cloud-init/artifacts/generateproxycerts.sh"
 	kubernetesCustomSearchDomainsScript      = "k8s/cloud-init/artifacts/setup-custom-search-domains.sh"
 	kubeletSystemdService                    = "k8s/cloud-init/artifacts/kubelet.service"
-	kmsSystemdService                        = "k8s/cloud-init/artifacts/kms.service"
```

Comment:
We should be able to delete the `parts/k8s/cloud-init/artifacts/kms.service` file entirely from the codebase now

Reply:
deleted `parts/k8s/cloud-init/artifacts/kms.service`